packages/docs/docs/careplans/clinical-decision-support.md

```diff
@@ -0,0 +1,71 @@
+
+### Training Data
+
+Demonstrating which training data was used to train an algorithm, and keeping a record of the versioning on said data is part of the certification process. In the context of a Medplum implementation, be prepared to keep all of your training data in a [Medplum project](/docs/auth/user-management-guide#background-user-model) which will show which data was used to train the models and that the data is updated (feedback loops).
+
+### Code Systems
+
+It is recommended that data is tagged with UMLS code systems, including LOINC, SNOMED and RxNorm. For certification, data must conform to [USCDI profiles](/docs/fhir-datastore/understanding-uscdi-dataclasses). Implementations of predictive clinical decision support sometimes include annotating data with code systems using an algorithm or large language model.
+
+### Insights Reporting
+
+Electronic health records that support predictive clinical decision support will be required to report on the usage of their product. Prepare the following basic statistics as part of certification: the number of times the decision support was used, how many unique clinicians used it, the number of times it was updated and any complaints received.
+
+## Linked Referential
+
+Linked referential clinical decision support are hyperlinks that link to reference material that is specific to the clinical context of a specific patient or population. The ONC criteria that define this standard are 170.205(a)(3,4) amd relate to the retrieving context-aware knowledge using the HL7 Infobutton.
+
+'''caution
+Medplum is not certified for (a)(3,4) but serves as a basis for those who wish to implement. The Clinical Profile and [Diagnostic Report](https://storybook.medplum.com/?path=/story/medplum-diagnosticreportdisplay--simple) react components serve as common launch points for Infobutton implementations.
+'''
+
+To support the linked referential clinical decision support the system should be capable of retrieving information based on one or more of the following data elements. The Clinical Profile react component highlights the data elements.
+
+TODO: Link to the [Clinical Profile](https://github.com/medplum/medplum/issues/3625) react component
+
+- Demographic information
+- Problems list (Conditions in FHIR)
+- Medications
+- Smoking status
+
+[UpToDate](https://www.wolterskluwer.com/en/solutions/uptodate/uptodate-advanced/workflow-integration) is a common provider for linked referential data.
+
+## Evidence Based
+
+Evidence-based clinical decision support systems are largely related to medication administration, drug interactions (with other drugs, foods, OTC medications, etc.) and dosing. Evidence-based clinical decision support systems plug into health record systems via SMART-App-Launch links, iFrames or APIs, all of which are supported by Medplum.
+
+TODO: Link to the the [Smart-App-Launch react component](https://github.com/medplum/medplum/issues/3605)
```

Comment:
Let's remove this, but add a comment to issue #3605 to update this doc when the component is finished?